src/Elliptic/Actions/InitializeAnalyticSolution.hpp

```diff
@@ -32,47 +32,53 @@ namespace Actions {
 /*!
```

Comment:
This change has nothing to do with the rest of the PR and needs to be factored out. (so do many of the other commits as far as I can tell)